zhaquirks/xiaomi/aqara/roller_curtain_e1.py

```diff
@@ -69,36 +60,28 @@ class XiaomiAqaraRollerE1(XiaomiCluster, ManufacturerSpecificCluster):
 class AnalogOutputRollerE1(CustomCluster, AnalogOutput):
     """Analog output cluster, only used to relay current_value to WindowCovering."""
 
-    cluster_id = AnalogOutput.cluster_id
-
-    def __init__(self, *args: Any, **kwargs: Any) -> None:
-        """Init."""
-        super().__init__(*args, **kwargs)
-
-        self._update_attribute(0x0041, float(0x064))  # max_present_value
```

Comment:
@schwickster Sorry to ping you again here, but do you remember why you used `float(0x064)` instead of just `100.0` here? It should do the same, right? (or is there some weird behavior?)